pwndbg/gdblib/disasm/__init__.py

```diff
@@ -423,7 +425,7 @@ def near(
 
         # Handle visual splits in the disasm view
         # The second check here handles instructions like x86 `REP` that repeat the instruction
-        if insn.jump_like or insn.next == insn.address:
+        if insn.has_jump_target or insn.next == insn.address:
```

Comment:
Minor change that results in instructions in Arm such as 
```
pop pc
ldr pc, [some_address]
```
To have split in disasm view